test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+ENV["RAILS_ENV"] ||= "test"
+require_relative "../config/environment"
+require "rails/test_help"
+
+class ActiveSupport::TestCase
```

Comment:
If RSpec is installed, do you need Minitest?

Reply:
Is this whole file linked to miniTest? Would I need to remove also `test/application_system_test_case.rb` and `test/channels/application_cable/connection_test.rb`?

Reply:
Yep, if you decided to use RSpec, then the whole `test` directory can be deleted.

Reply:
Cool, thank you G. I'm going to delete all unused folders/files in PR for issue 3.